pkg/rtc/participant.go

```diff
@@ -388,7 +389,7 @@ func (p *ParticipantImpl) OnClaimsChanged(callback func(types.LocalParticipant))
 
 // HandleOffer an offer from remote participant, used when clients make the initial connection
 func (p *ParticipantImpl) HandleOffer(offer webrtc.SessionDescription) error {
-	p.params.Logger.Infow("received offer", "transport", livekit.SignalTarget_PUBLISHER)
+	p.params.Logger.Infow("received offer", "transport", livekit.SignalTarget_PUBLISHER, "sdp", offer.SDP)
```

Comment:
Had removed the SDP as DZ felt it will be spammy. Maybe, remove it after some time of soaking this?

Reply:
will remove it now, used for debug